app/templates/profile.hbs

```diff
@@ -1,13 +1,27 @@
 <div class="div-outer-white">
+
+ <div class="div-upper-blue">
+ </div>
+
   <header>
+    <div class="div-toolbar">
+    <button class="edit-button" {{action 'editProfile'}}>Edit</button>
+    Profile
```

Comment:
Its really nice to wrap "floating"words like this in a tag. In my experience, CSS plays nicer if it can have a "solid" tag to grab ahold of instead of just text.